test/test_file_processor.py

```diff
@@ -72,6 +83,73 @@ def test_load_csv_with_cp932(self):
                 pd.testing.assert_frame_equal(df, loaded_df)
 
 
+target_orients = [None, 'records']
+
+
+class TestJsonFileProcessor(unittest.TestCase):
+    @parameterized.expand([(orient,) for orient in target_orients])
+    def test_dump_and_load_dataframe(self, orient):
```

Comment:
[imo]
These tests seem to check if dumped files can be loaded as same dataframe.
I suggest that you check the dumped data have expected format to avoid this error https://github.com/m3dev/gokart/pull/458/files#r1993450860

```python
expected_json = """{'foo': 1, 'bar': 2}
{'foo': 1, 'bar': 2}
"""
with local_target.open('r') as f:
  actual_json = f.read()

assert expected_json == actual_json
loaded_df = processor.load(actual_json)
pd.testing.assert_frame_equal(df, loaded_df)
```